dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/strategy/FileAssetTransformStrategy.java

```diff
@@ -0,0 +1,73 @@
+package com.dotmarketing.portlets.contentlet.transform.strategy;
+
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.*;
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.DESCRIPTION;
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.FILE_NAME_FIELD;
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.META_DATA_FIELD;
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.MIMETYPE_FIELD;
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.TITLE_FIELD;
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.UNDERLYING_FILENAME;
+import static com.dotmarketing.util.UtilHTML.getIconClass;
+import static com.dotmarketing.util.UtilHTML.getStatusIcons;
+import static com.dotmarketing.util.UtilMethods.getFileExtension;
+import static com.dotmarketing.util.UtilMethods.isNotSet;
+import static com.dotmarketing.util.UtilMethods.isSet;
+import static com.liferay.util.StringPool.BLANK;
+
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.fileassets.business.FileAsset;
+import com.dotmarketing.portlets.fileassets.business.FileAssetAPI;
+import java.util.Map;
+import java.util.Set;
+
+public class FileAssetTransformStrategy extends WebAssetStrategy<FileAsset> {
+
+    FileAssetTransformStrategy(final TransformToolBox toolBox) {
+        super(toolBox);
+    }
+
+    @Override
+    public FileAsset fromContentlet(final Contentlet contentlet) {
+        return toolBox.fileAssetAPI.fromContentlet(contentlet);
+    }
+
+    @Override
+    public Map<String, Object> transform(final FileAsset fileAsset, final Map<String, Object> map,
+            Set<TransformOptions> options)
+            throws DotDataException, DotSecurityException {
+        final String title = (String)map.get(TITLE_FIELD);
+        if(isNotSet(title)){
+            final Identifier identifier = toolBox.identifierAPI.find(fileAsset.getIdentifier());
+            map.put(TITLE_FIELD, identifier.getAssetName());
+        }
+        map.put(MIMETYPE_FIELD, fileAsset.getMimeType());
+        map.put("name", fileAsset.getFileName());
+        map.put("size", fileAsset.getFileSize());
+        final String description = fileAsset.getStringProperty(FileAssetAPI.DESCRIPTION);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'FileAssetAPI.DESCRIPTION' due to existing static import 'com.dotmarketing.portlets.fileassets.business.FileAssetAPI.DESCRIPTION'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)